diff --git a/setup.py b/setup.py
index e74e6d97..4e1eef1c 100644
--- a/setup.py
+++ b/setup.py
@@ -584,11 +584,10 @@ class my_build_ext(build_ext):
 
     def _why_cant_build_extension(self, ext):
         # Return None, or a reason it can't be built.
-        # Exclude exchange 32-bit utility libraries from 64-bit
-        # builds. Note that the exchange module now builds, but only
-        # includes interfaces for 64-bit builds.
-        if self.plat_name in ["win-amd64", "win-arm64"] and ext.name == "exchdapi":
-            return "No 64-bit library for utility functions available."
+        if ext.name in {'exchdapi', 'exchange'}:
+            # Dont need these modules, and they require headers not present in
+            # the windows SDK
+            return ext.name + " module is disabled"
         if ext.name == "exchdapi":
             return "Haven't worked out how to build on vs2015"
         # axdebug fails to build on 3.11 due to Python "frame" objects changing.
